docs/samples/python/example_pdf_annotation.ipynb

```diff
@@ -0,0 +1,254 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Annotating PII in a PDF\n",
+    "\n",
+    "This sample takes a PDF as an input, extracts the text, identifies PII using Presidio and annotates the PII using highlight annotations.\n",
+    "\n",
+    "## Prerequisites\n",
+    "Before getting started, make sure the following packages are installed. For detailed documentation, see the [installation docs](https://microsoft.github.io/presidio/installation).\n",
+    "\n",
+    "Install from PyPI:\n",
+    "```bash\n",
+    "pip install presidio_analyzer\n",
+    "pip install presidio_anonymizer\n",
+    "pip install presidio_image_redactor\n",
```

Comment:
Is the Presidio Image Redactor used here?

Reply:
Good spot! It's a copy and paste error from another sample.